index.md

```diff
@@ -41,14 +32,20 @@ the things you need to make the magical 'end-to-end' micro:bit experience.
 If you'd like to help us make the micro:bit better, then there are a number of
 ways that you can get involved
 
-* Contribute features and fixes to one of the upstream projects that the micro:bit relies upon,
-such as the micro:bit Runtime, Makecode, the Python editor, or mbed. These projects are
-explained in this site.
-* Help us build a brilliant hardware ecosystem around micro:bit - derivative boards,
+## Code
+Contribute features and fixes to one of the upstream projects that the micro:bit relies upon:
+  * the micro:bit [Device Abstraction Layer(DAL/runtime)](https://lancaster-university.github.io/microbit-docs/)
```

Comment:
The other bullet points start with a capital letter, so this should probably do as well.
Also, don't know if this is necessary, but I'd add an space before the brackets.

```suggestion
  * The micro:bit [Device Abstraction Layer (DAL/runtime)](https://lancaster-university.github.io/microbit-docs/)
```